src/containers/Landing/index.tsx

```diff
@@ -1,18 +1,22 @@
 import { styled } from '@mui/material/styles';
+import { Balance } from '~/containers';
+import { Allownace } from '~/containers/Allowance';
```

Comment:
it would be better to group these imports
```suggestion
import { Balance } from '~/containers';
import { Allownace } from '~/containers/Allowance';
```

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/1/commits/0dc604a0b10a97efd7632a4b52f79675777e2388